nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -160,6 +160,10 @@
 
 - `getent` has been moved from `glibc`'s `bin` output to its own dedicated output, reducing closure size for many dependents. Dependents using the `getent` alias should not be affected; others should move from using `glibc.bin` or `getBin glibc` to `getent` (which also improves compatibility with non-glibc platforms).
 
+- `maintainers/scripts/update-luarocks-packages` is now a proper package
+  `luarocks-packages-updater` that can be run to maintain out-of-tree luarocks
+  packages
+
```

Comment:
question: Is a release notes entry necessary for something like this? To me, it feels more internal, and less interesting to NixOS users.

Reply:
it might be the wrong section but I think it's worth reporting, like when the vim plugins updater got changed, you got surprised, you got surprised it disappeared (I think even fewer people use the luarocks updater but I am often surprised by people using code I didnt think anyone would use).